### DISCUSSION TITLE
error lab 02.02 task 3

### DISCUSSION THREAD

**Discussion by dbertelm** [rramosp/ai4eng.v1#137] [02 - PYTHON]
buenas, al tratar de hacer submit en esta tarea me genera un error.
el cual es el siguiente: error: ERROR EXECUTING YOUR CODE
pero en la comprobacion manual lo ejecuta sin problemas y me da el resultado.

**Comment by dbertelm** [comment]
tambien me aparece el mismo error en la task 4 de este mismo laboratorio

**Comment by rramosp** [comment]
hola dbertelm, tienes que adjuntarnos los mensajes de error ...

On Sun, Jun 12, 2022 at 6:48 PM dbertelm ***@***.***> wrote:

> tambien me aparece el mismo error en la task 4 de este mismo laboratorio
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/137#discussioncomment-2934579>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIMMRGYOQRWVLZKK27TVOZZNFANCNFSM5YSPR4QQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
